README.md

```diff
@@ -136,6 +136,11 @@ By default, we do not show days from the previous month and the next month in th
    initialVisibleMonth: PropTypes.func,
 ```
 
+Optionally you can provide calendar caption block renderer.
+```js
+   renderCalendarCaption: PropTypes.func,
```

Comment:
I'm concerned with the nomenclature of using `caption` here as we already have a `<caption>` dom element on each table inside each `CalendarMonth` component. I think using `Calendar` is also somewhat problematic as this is rendered on the `DayPicker` level not on the `CalendarMonth` level. Perhaps renaming it to `renderDayPickerCaption` or `renderDayPickerAdditionalInfo` might be a good idea? What do you think?